src/app/core/component.ts

```diff
@@ -56,7 +56,7 @@ export default class Component {
         // Create and set the ports here itself
 
         const cleanparamdata = this._params.parameters;
-        if (mint !== "" || mint === null) {
+        if ((mint !== "" && mint !== "TEXT") || mint === null) {
```

Comment:
This should throw an error !

Reply:
Sorry ! I misread the code. We need to clean up this logic. I'm not sure what this is doing right now. I feel like a bunch of errors are just gonna creep through this

Reply:
The null portion may be unnecessary; it and the empty string were there prior to my edits